### PR TITLE
Removal of the Option to proceed page, some text updates and removal of redundant code

### PR DESCRIPTION
**Pull request by richard-mohammed-def**
This PR includes text changes, the removal of the Option to proceed page and removal of redundant code for pages that are no longer visited throughout the flow.

Useful links for context:
https://trello.com/c/u7VazJWk/34-remove-do-you-wish-to-proceed-page
https://trello.com/c/sAXWUxOQ/51-tell-us-about-your-item

https://p972ra.axshare.com/

### CODE REVIEW

File: server/routes/eligibility-checker/cannot-continue.route.js [hunk 1]
```diff
@@ -40,21 +37,11 @@ const handlers = {
...
-    helpText1a: `To use this service, you must know for sure whether your item ${
-      containsElephantIvoryIdk
-        ? 'contains elephant ivory.'
-        : 'qualifies for exemption.'
-    }`,
-    helpText1b: containsElephantIvoryIdk
-      ? 'If you’re uncertain about your item and you choose to declare it anyway, we’ll assume it does contain elephant ivory.'
-      : '',
+    helpText1a: 'To use this service, you must know for sure whether your item qualifies for exemption.',
+    helpText1b: 'If you’re uncertain about your item and you choose to declare it anyway, we’ll assume it does contain elephant ivory.',
```
Comment (ben-sagar) [inline server/routes/eligibility-checker/cannot-continue.route.js:44]: If I'm reading this correctly, this route returns entirely static content now so it should just sit in the view file

Comment (richard-mohammed-def) [reply server/routes/eligibility-checker/cannot-continue.route.js:44]: Yeah. That make sense. I'll make the change

Comment (richard-mohammed-def) [reply server/routes/eligibility-checker/cannot-continue.route.js]: @ben-sagar I've updated this now. Everything, apart from the pageTitle (which is used for Analytics), moved to the view

**Review by ben-sagar — APPROVED** [review]
I do like to see unnecessary code being removed! 👍

**Pull request merged**